dict/P/pandunia.tsv

```diff
@@ -562,6 +566,7 @@ PWN:discipline.n.01		lojia	loj'ia
 PWN:disinter.v.01		exhume	ex.hum'e	
 PWN:disk.n.01		diske	disk'e	eng:tur:disk, fra:disque, spa:por:disco, rus:диск (disk), hin:डिस्क (ḍisk), ben:ডিস্ক (ḑisk), jpn:ディスク (disuku), kor:디스크 (diseukeu)
 PWN:diskette.n.01		diskit	disk·it	
+PWN:disparate.s.01		dispartet	dis.part·et	

```

Comment:
_dis.part·et_ would mean 'parted asunder', see https://en.wiktionary.org/wiki/dispart

This would should be _dis.par_. (I just added _par_ back from old Pandunia data sheet.)